pages/index.vue

```diff
@@ -31,14 +33,10 @@ main {
 
 main > * + *,
 footer {
-  margin-top: 6rem;
+  margin-top: 10rem;
 
   @media (--viewport-medium) {
```

Comment:
Check if we need to keep a lower `margin-top` for `--viewport-small`.

Reply:
`--viewport-medium` will also work for smaller devices. So basically `--viewport-medium` is `width < 1024px`.

Can you check if the spacing looks large in your mobile device, we can reduce it using `--viewport-small`. In designs atleast they have kept same spacing as desktop but it was very large in my mobile device, hence I reduced it